DESCRIPTION

```diff
@@ -0,0 +1,25 @@
+Package: acquacotta
+Type: Package
+Title: R Shiny App to Display Run Reports Generated by the OICR SeqWare Pipeline
+Version: 0.1.0
+Author: Savo Lazic
+Author@R: person("Savo", "Lazic", email = "savo.lazic@oicr.on.ca", role = c("aut", "cre"))
+Maintainer: Savo Lazic <savo.lazic@oicr.on.ca>
+Description: R Shiny App to Display Run Reports Generated by the OICR SeqWare Pipeline
```

Comment:
@apmasell @morgantaschuk is "SeqWare Pipeline" going to become obsolete once Niassa is fully operational? Might be better to just say "OICR Sequencing Pipeline" or "OICR Genome Sequence Informatics Pipeline". 

(Sentence case would be more informative than title case here, but that's a weakly held conviction :) )

Reply:
Niassa is fully operational, so removing the name 'SeqWare' is :+1:

Reply:
Suggestions have been incorporated.